lux/vislib/matplotlib/Histogram.py

```diff
@@ -59,12 +59,15 @@ def initialize_chart(self):
 
         x_label = ""
         y_label = ""
+        axis_title = f"{msr_attr.attribute} (binned)"
```

Comment:
Should we be using `msr_attr_abv = str(msr_attr_attribute)` like we do in Altair, or is that unnecessary?

Reply:
good catch!